app/bundles/UserBundle/Tests/Controller/UserControllerTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
```

Comment:
Declare strict types for new files
```suggestion

declare(strict_types=1);

```

Reply:
Done